docs/user/commandline_old_commands.rst

```diff
@@ -0,0 +1,72 @@
+Old Command Line Commands
```

Comment:
This page won't be included unless it's referenced in user/index.rst (perhaps as an appendix).